src/composables/Project.js

```diff
@@ -589,3 +589,19 @@ export async function gitCommit(projectId, message) {
     message,
   });
 }
+
+/**
+ * Get all logs from git log.
+ * @param {String} projectId - Id of project.
+ * @param {String} ref - Ref to make the git log.
+ * @param {Number} depth - Number of log to retrieve
```

Comment:
missing `.`

Reply:
why not `@param {Number} [depth=10] - Number of log to retrieve.` ?

Reply:
done